Login.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
```

Comment:
@oneamanish Could you please lmk what you did here? 
The application is on reactjs, firebase and you have added a HTML file here